src/windows/mainwindow.cpp

```diff
@@ -57,6 +73,7 @@ void MainWindow::on_actionLogout_triggered()
     Login::requestLogin();
 }
 
+/*Swich views for stacks in mainView within the mainWindow page (page 1) in centralStack*/
```

Comment:
This will kill me later on. Add some spaces between the `*` and the words
```suggestion
/* Swich views for stacks in mainView within the mainWindow page (page 1) in centralStack */
```